### PR TITLE
Add news items

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Add eight news items.

### CODE REVIEW

**Comment by jburel** [comment]
Looking at the file names that are currently merged and those ones
it should be `` name-x-y-z.md``
cc @hflynn

**Comment by hflynn** [comment]
I've removed all the 'Dear All' and 'Regards' bits so they don't read like emails any more and the old community page links can be updated as internal page links to ({{ site.baseurl }}/support/)

**Comment by sbesson** [comment]
@rleigh-codelibre can you minimally push the changes to remove `Dear all/Regards` from the existing posts. We can address the other comments in follow-up cleanup PRs.

Also are you combining 63fc372 and the third checklist as part of this PR or is it a separate one?

**Comment by rleigh-codelibre** [comment]
I'll do it all on this PR.  This afternoon hopfully.

File: _posts/2016-02-29-omero-5-2-2.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO 5.2.2
+intro-blurb: The OME team are pleased to announce the release of OMERO 5.2.2
```
Comment (jburel) [inline _posts/2016-02-29-omero-5-2-2.md]: ``is`` instead of ``are``

File: _posts/2016-04-11-ome-files-c++-0-1-0.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: Release of OME-Files-C++ 0.1.0
+intro-blurb: The OME team are pleased to announce the release of OME-Files-C++ 0.1.0
```
Comment (jburel) [inline _posts/2016-04-11-ome-files-c++-0-1-0.md]: same

File: _posts/2016-04-18-bio-formats-5-1-9.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Release of Bio-Formats 5.1.9
+intro-blurb: The OME team are pleased to announce the release of Bio-Formats 5.1.9
```
Comment (jburel) [inline _posts/2016-04-18-bio-formats-5-1-9.md]: same

File: _posts/2016-05-30-omero-5-2-4.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Security release of OMERO 5.2.4
+intro-blurb: The OME team are pleased to announce the release of OMERO 5.2.4
```
Comment (jburel) [inline _posts/2016-05-30-omero-5-2-4.md]: same

File: _posts/2016-08-01-ome-files-c++-0-2-0.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: Release of OME Files C++ 0.2.0
+intro-blurb: The OME team are pleased to announce the release of OME Files C++ 0.2.0
```
Comment (jburel) [inline _posts/2016-08-01-ome-files-c++-0-2-0.md]: same

File: _posts/2016-08-02-omero-5-2-5.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Security release of OMERO 5.2.5
+intro-blurb: The OME team are pleased to announce the release of OMERO 5.2.5
```
Comment (jburel) [inline _posts/2016-08-02-omero-5-2-5.md]: same

File: _posts/2016-08-02-omero-5-2-5.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+---
+Today we are releasing OMERO 5.2.5. This is a security release to fix
+the OMERO.shares API which was bypassing intended security
+restrictions and granting elevated privileges to read
+otherwise-restricted data.
+
+Full details of the issue are available
+[here](https://www.openmicroscopy.org/site/products/omero/secvuln/2016-SV2-share).
```
Comment (jburel) [inline _posts/2016-08-02-omero-5-2-5.md]: replace ``here`` by ``2016-SV2-share`` to match the other secvuln url

File: _posts/2016-08-02-omero-5-2-5.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+This release **does not** upgrade the version of Bio-Formats which
+OMERO uses.
+
+The software is available [here](http://downloads.openmicroscopy.org/omero/5.2.5).
```
Comment (jburel) [inline _posts/2016-08-02-omero-5-2-5.md]: replace ``here`` by  ``archived downloads``

File: _posts/2016-08-02-omero-5-2-5.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+Upgrade information is
+[here](http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-upgrade.html).
```
Comment (jburel) [inline _posts/2016-08-02-omero-5-2-5.md]: Replace by
``Upgrade information is in the [server upgrade documentation](url)``

File: _posts/2016-08-18-bio-formats-5-2-0.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Release of Bio-Formats 5.2.0
+intro-blurb: The OME team are pleased to announce the release of Bio-Formats 5.2.0
```
Comment (jburel) [inline _posts/2016-08-18-bio-formats-5-2-0.md]: same

File: _posts/2016-08-18-bio-formats-5-2-0.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+-  licensing updates for XSL transforms, specification code and xsd-fu Python code (to Simplified 2-clause BSD)
+-  JACE C++ implementation has been decoupled and [marked as legacy](https://github.com/ome/bio-formats-jace)
+-  Bio-Formats C++ native implementation has been decoupled and is now being [released as OME Files C++](http://downloads.openmicroscopy.org/ome-files-cpp/)
+
+Full details of all of these changes are available on the Bio-Formats
+[version history
+page](http://www.openmicroscopy.org/site/support/bio-formats5.2/about/whats-new.html).
+
+Bio-Formats 5.2.0 is available to download from
+[here](http://downloads.openmicroscopy.org/bio-formats/5.2.0/)
```
Comment (jburel) [inline _posts/2016-08-18-bio-formats-5-2-0.md]: change ``here``

**Comment by sbesson** [comment]
@rleigh-codelibre can you push the changes suggested above?

File: _posts/2014-04-07-omero-bio-formats-5-0-1.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+releases. Therefore, if you are having an issue with 5.0.0 which has
+not yet been addressed, be assured that 5.0.2 will be following with
+further fixes as soon as possible. Note that only the jar-signing
+issue will be back-ported to the 4.4.x line - we will be releasing
+4.4.11 imminently with this fix included.
+
+For OMERO, this release includes:
+
+-  Code signing to fix the Java Web Start issues
+-  New import scenarios [documentation](http://www.openmicroscopy.org/site/support/omero5/sysadmins/index.html#data-import-options) including for the new 'in-place' import capability
```
Comment (sbesson) [inline _posts/2014-04-07-omero-bio-formats-5-0-1.md]: :+1: once this PR is merged, I can certainly use https://github.com/openmicroscopy/www.openmicroscopy.org/pull/77 to:

- update all the recent doc links to use the new URL
- review the remaining `site/support` URLs

File: _posts/2014-05-28-omero-bio-formats-5-0-2.md [hunk 13]
```diff
@@ -24,16 +24,17 @@ Bio-Formats improvements include:
...
 - Downloading original files from the command line;
 - Improvements to the way OMERO.matlab handles annotations;
 - Improvements to tagging workflows in the clients;
-- Further work on [code signing](http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-webstart-codesigning.html) for webstart and accompanying documentation;
+- Further work on [code signing](http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-webstart-codesigning.html)
+  for webstart and accompanying documentation;
 - Disk space usage reporting for OMERO.web admins;
 - Custom webstart intro page templates;
-- Enabling searching by Image ID in the clients.
+- Enabling searching by Image ID in the clients.;
```
Comment (jburel) [inline _posts/2014-05-28-omero-bio-formats-5-0-2.md]: why ``.;``?

Comment (rleigh-codelibre) [inline _posts/2014-05-28-omero-bio-formats-5-0-2.md]: Mistake after fixing the wrapping; should they not all be semicolons?  I.e. should I remove the period?

Comment (jburel) [inline _posts/2014-05-28-omero-bio-formats-5-0-2.md]: not an expert but I will remove the ``.``

**Comment by sbesson** [comment]
Last comment can be addressed in the cleanup. Merging.

**Pull request merged**